### PR TITLE
Add token SnBNB

### PR DESCRIPTION
**Pull request by Leonardohhh**
Add SnBNB token to models/tokens/bnb/tokens_bnb_bep20.sql
Add SnBNB token to models/prices/bnb/prices_bnb_tokens.sql

### CODE REVIEW

**Comment by Hosuke** [comment]
Hi @Leonardohhh, thank you for your contribution.
Please sign the CLA so we can continue the PR.

> **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/duneanalytics/spellbook/blob/main/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> 
> I have read the CLA Document and I hereby sign the CLA
> 
> You can retrigger this bot by commenting **recheck** in this Pull Request

**Comment by Leonardohhh** [comment]
I have read the CLA Document and I hereby sign the CLA

File: models/prices/bnb/prices_bnb_tokens.sql [hunk 1]
```diff
@@ -204,7 +204,8 @@ FROM
     ('id2-space-id', 'bnb', 'ID', 0x2dff88a56767223a5529ea5960da7a3f5f766406, 18),
     ('ankrbnb-ankr-staked-bnb', 'bnb', 'ankrBNB', 0x52F24a5e03aee338Da5fd9Df68D2b6FAe1178827, 18),
     ('joyg-kismet-joy-gold', 'bnb', 'JOYG', 0xe63c37e50a045c247b5003e24ba066f1cc3f0ff3, 8),
-    ('gft-galaxy-finance', 'bnb', 'GFT', 0xe77932b1216125848e82c3967e75698362168f99, 18)
+    ('gft-galaxy-finance', 'bnb', 'GFT', 0xe77932b1216125848e82c3967e75698362168f99, 18),
+    ('snbnb-synclub-staked-bnb', 'bnb', 'SnBNB', 0xb0b84d294e0c75a6abe60171b70edeb2efd14a1b, 18)
```
Comment (Hosuke) [inline models/prices/bnb/prices_bnb_tokens.sql:208]: API: https://api.coinpaprika.com/v1/coins/snbnb-synclub-staked-bnb

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @Leonardohhh

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**